### PR TITLE
Process CC Compiler params in the execution phase

### PR DESCRIPTION
**Pull request by brentleyjones**
This reduces the amount of Starlark retained memory.

### CODE REVIEW

File: examples/integration/test/fixtures/bwx.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -8817,9 +8817,9 @@
 			files = (
 				A7EE330C67CB794EA5DC8FCE /* Assets.xcassets in Resources */,
 				96536BFC80D7064031732BC1 /* bucket in Resources */,
-				57B6669C9AE0F887A3D54BDA /* bucket in Resources */,
 				EB509AB2045BC989DBE6563C /* bucket in Resources */,
 				130B4EF21F478D0D76C741D0 /* bucket in Resources */,
+				57B6669C9AE0F887A3D54BDA /* bucket in Resources */,
```
Comment (brentleyjones) [inline examples/integration/test/fixtures/bwx.xcodeproj/project.pbxproj:8822]: I'm hoping this is just a left over change that didn't get picked up somehow, and that the ordering is now stable...

File: examples/integration/test/fixtures/bwb.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -11912,6 +11906,7 @@
 				DEPLOYMENT_LOCATION = "$(INDEXING_DEPLOYMENT_LOCATION__$(INDEX_ENABLE_BUILD_ARENA)),";
 				DSTROOT = "$(PROJECT_TEMP_DIR)";
 				ENABLE_DEFAULT_SEARCH_PATHS = NO;
+				GCC_OPTIMIZATION_LEVEL = 0;
```
Comment (brentleyjones) [inline examples/integration/test/fixtures/bwb.xcodeproj/project.pbxproj:11909]: `clang` default is `-O0`, but Xcode default is `-Os`, so we need to override this to allow the CC params files to work correctly.

**Comment by brentleyjones** [comment]
Seems like it's a huge memory savings.

**Review by thiagohmcruz — APPROVED** [review]

**Comment by brentleyjones** [comment]
Merging to unblock other changes.

**Pull request merged**